reference/src/glossary.md

```diff
@@ -73,6 +81,43 @@ niches. For example, the "all bits uninitialized" is an invalid bit-pattern for
 `&mut T`, but this bit-pattern cannot be used by layout optimizations, and is not a
 niche.
 
+#### Call ABI
+
+The *call ABI* determines how a type is passed *by value* across a function boundary.
+
+Note: The set of possible call ABIs is not stable. Currently, it consists of:
+
+```rust,ignore
+enum Abi {
+    Uninhabited,
+    Scalar(Scalar),
```

Comment:
`ScalarPair` is missing.

Reply:
That's intentional. ScalarPair is an implementation detail useful for some optimizations that we don't want to guarantee it exists - right @eddyb ?

Reply:
Yeah, it's just an aggregate that we pass around as two scalars *only* in Rust ABIs and within functions.